enos/.gitignore

```diff
@@ -0,0 +1 @@
+.enos/
```

Comment:
Heads up that it looks like you've got an editor configuration issue where files aren't getting their trailing EOL on save.